docker-compose-pg-load.yml

```diff
@@ -7,13 +7,10 @@ services:
     image: ${POSTGRES_IMAGE:-perconalab/percona-distribution-postgresql:13.3}
     container_name: pmm-agent-postgres-pgmonitor
     command: >
-      -c shared_preload_libraries=pg_stat_monitor,pg_stat_statements
```

Comment:
so we only support pg_stat_monitor?

Reply:
This is only used for generating some metrics in pg_stat_monitor, so statements is not required here.

Reply:
Good question

Reply:
It is another compose used for generating metrics, in original one we still have statements.